Mathlib/Algebra/Order/Group/WithTop.lean

```diff
@@ -51,9 +51,6 @@ theorem coe_neg (a : α) : ((-a : α) : WithTop α) = -a :=
   rfl
 #align with_top.coe_neg WithTop.LinearOrderedAddCommGroup.coe_neg
 
-@[simp]
-theorem neg_top : -(⊤ : WithTop α) = ⊤ := rfl
```

Comment:
I think you should keep this one since it's eligible for dsimp

Reply:
Does the simpNF linter work for `dsimp` now? I saw there was some talk about that in Zulip, and previously this lemma triggered it.

Reply:
I'm not quite sure whether it's there just yet (and you will have to merge master), but you can always tag this lemma with `nolint simpNF`

Reply:
Alright, I tagged it with `nolint simpNF`